testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractSinglenodeLdap.java

```diff
@@ -93,5 +93,10 @@ protected String getBaseImage()
         return "centos6-oj8-openldap";
     }
 
+    protected String getConfigFilePath()
```

Comment:
You don't need to make it abstract. You can simply copy the file once again to override singlenode-ldap config file.